CHANGELOG.md

```diff
@@ -6,6 +6,37 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+
+- **CUMULUS-1684**
+  - In previous releases, a provider's username and password were encrypted
+    using a custom encryption library. That has now been updated to use KMS.
+    This release includes a Lambda function named
+    `<prefix>-ProviderSecretsMigration`, which will re-encrypt existing
+    provider credentials to use KMS. After this release has been deployed, you
+    will need to manually invoke that Lambda function using either the AWS CLI
+    or AWS Console. It should only need to be run once.
+  - Future releases of Cumulus will invoke a
```

Comment:
These first two points should really be under a "Please note" or "Migration steps" header

Reply:
Fixed in fc0dbd4d7